### PR TITLE
[1.0] pin aioftp version for python 3.6

### PR DESCRIPTION
**Pull request by Cadair**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,8 @@ requirements:
     - setuptools
     - tqdm
     - aiohttp
-    - aioftp
+    - aioftp<=0.16.1 # [py=36]
```
Comment (nabobalis) [inline recipe/meta.yaml]:
```suggestion
    - aioftp <=0.16.1  # [py=36]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,7 +24,8 @@ requirements:
...
+    - aioftp # [py>36]
```
Comment (nabobalis) [inline recipe/meta.yaml]:
```suggestion
    - aioftp  # [py>36]
```

**Comment by Cadair** [comment]
> noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python.

crap

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 90dfb05d0a65d00669f5245371a4b33cbebcb5cfba86f4aad7f956984394eefd
 
 build:
-  number: 0
+  number: 1
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
   noarch: python
```
Comment (nabobalis) [inline recipe/meta.yaml:15]:
```suggestion
```

**Comment by nabobalis** [comment]
Will need just relinting after removing noarch.

**Comment by Cadair** [comment]
@conda-forge-admin, please rerender

**Review by nabobalis — APPROVED** [review]

**Comment by Cadair** [comment]
@conda-forge-admin, please rerender

**Comment by Cadair** [comment]
or maybe

@nabobalis, please rerender

**Comment by Cadair** [comment]
it did it by itself! good bot

**Review by nabobalis — APPROVED** [review]

**Pull request merged**